### PR TITLE
Tickets 19 20

### PR DESCRIPTION
**Pull request by cecerodrgz**
The createRoot is been funky, I already tried to fix it but it is not cooperating with me. I'd appreciate if anyone could help me with it, I haven't used it before. 
This is on the Date Of Birth file, by the way!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -69,26 +69,32 @@ const App = () => {
...
+                        element={
+                            //Uncomment protected route when authentication is done
 
-  );
+                            // <ProtectedRoute requiredAuthLevel='anonymous'>
+                            <LoginPage />
+                            // </ProtectedRoute>
+                        }
+                    />
+                    <Route path="/sign-up" element={<SignUp />} />
```
Comment (Ekaterina-Bondareva) [inline src/App.js:92]: According to the ticket this route should be inside `ProtectedRoute` with `requiredAuthLevel `set to `anonymous`. Could you please fix?
`Route should be a protected route (use ProtectedRoute component) with a requiredAuthLevel of 'anonymous'.`

File: src/components/SignUpPage/DoB.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  return (
+    <div>
+      <Form onSubmit={handleSubmit}>
+        <Form.Group controlId="DateOfBirth">
+          <Form.Label>Date of Birth</Form.Label>
+
+          <DatePicker
+            selected={values && values.date ? values.date : null}
+            onChange={(e) => {
+              setFieldValue('date', e);
```
Comment (Ekaterina-Bondareva) [inline src/components/SignUpPage/DoB.js:29]: When clicking on any date inside date picker I see the following error. Could you please fix?
```
DoB.js:29 Uncaught TypeError: setFieldValue is not a function
    at onChange (DoB.js:29:1)
    at r.setSelected (react-datepicker.min.js:1:1)
```

**Comment by Ekaterina-Bondareva** [comment]
Also I can not type any data in fields: "user name", "password" and "confirm password".

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (cecerodrgz) [reply src/App.js:92]: This still needs fixing, we went over it with Nikki and we will need it as is for now, the BE needs to do something about the protected route, but I am not sure what.

File: src/App.js [hunk 3]
```diff
@@ -69,26 +69,32 @@ const App = () => {
...
+                        element={
+                            //Uncomment protected route when authentication is done
 
-  );
+                            // <ProtectedRoute requiredAuthLevel='anonymous'>
+                            <NewLogin />
+                            // </ProtectedRoute>
+                        }
+                    />
+                    <Route path="/sign-up" element={<NewSignUp/>} />
```
Comment (jpremo) [inline src/App.js:92]: Can you add the commented out ProtectedRoute the same as you did for the NewLogin Page here?

File: src/components/LoginPage/LoginPage.js [hunk 4]
```diff
@@ -1,12 +1,99 @@
...
+                    textarea={false}
+                />
+
+                <TextInput
+                    placeholder="Confirm Password"
+                    type="password"
+                    id="logInConfirmPassword"
+                    label="Confirm Password"
+                    textarea={false}
+                />
```
Comment (jpremo) [inline src/components/LoginPage/LoginPage.js:40]: We don't need a confirm password for logging in. Can you remove this?

**Review by jpremo — CHANGES_REQUESTED** [review]
Nice work. A few smaller changes then it should be good to go

**Comment by jpremo** [comment]
Can you merge develop and resolve merge conflicts before requesting another review?

**Pull request closed without merging**